benchmarks/graphics_pipeline/GraphicsBenchmarkApp.h

```diff
@@ -152,33 +152,34 @@ class GraphicsBenchmarkApp
-    Texture                                                       mNormalMapTexture;
-    Texture                                                       mMetalRoughnessTexture;
-    Entity                                                        mSkyBox;
-    Entity                                                        mSphere;
-    Entity2D                                                      mFullscreenQuads;
-    bool                                                          mEnableMouseMovement = true;
-    std::vector<grfx::BufferPtr>                                  mDrawCallUniformBuffers;
-    std::array<grfx::GraphicsPipelinePtr, kPipelineCount>         mPipelines;
-    std::array<grfx::ShaderModulePtr, kAvailableVsShaders.size()> mVsShaders;
-    std::array<grfx::ShaderModulePtr, kAvailablePsShaders.size()> mPsShaders;
-    std::array<grfx::MeshPtr, kMeshCount>                         mSphereMeshes;
-    MultiDimensionalIndexer                                       mGraphicsPipelinesIndexer;
-    MultiDimensionalIndexer                                       mMeshesIndexer;
-    std::vector<LOD>                                              mSphereLODs;
+    std::vector<PerFrame>                                               mPerFrame;
+    FreeCamera                                                          mCamera;
+    float3                                                              mLightPosition = float3(10, 250, 10);
+    std::array<bool, TOTAL_KEY_COUNT>                                   mPressedKeys   = {0};
+    uint64_t                                                            mGpuWorkDuration;
+    grfx::ShaderModulePtr                                               mVSSkybox;
+    grfx::ShaderModulePtr                                               mPSSkybox;
+    grfx::ShaderModulePtr                                               mVSQuads;
+    Texture                                                             mSkyBoxTexture;
+    Texture                                                             mAlbedoTexture;
+    Texture                                                             mNormalMapTexture;
+    Texture                                                             mMetalRoughnessTexture;
+    Texture                                                             mQuadsTexture;
+    Entity                                                              mSkyBox;
+    Entity                                                              mSphere;
+    Entity2D                                                            mFullscreenQuads;
+    bool                                                                mEnableMouseMovement = true;
+    std::vector<grfx::BufferPtr>                                        mDrawCallUniformBuffers;
+    std::array<grfx::GraphicsPipelinePtr, kPipelineCount>               mPipelines;
+    std::array<grfx::GraphicsPipelinePtr, kFullscreenQuadsType.size()>  mQuadsPipelines;
```

Comment:
nit: this is a long block that's a bit hard to read. Perhaps we should split those into logical groups and add a comment? 
For ex:
```
// Full screen quads resources
std::array<grfx::GraphicsPipelinePtr, kFullscreenQuadsType.size()>  mQuadsPipelines;
...
```

Reply:
OK, split by skybox/sphere/quads.